controllers/posts.js

```diff
@@ -1,19 +1,19 @@
 const postsController = {
-    
-  };
-  
-module.exports = postsController;
+  logAllPostsRedirect:
+    ('../posts',
```

Comment:
Do we have to have the '../posts' path here? shouldn't be in the routes file?

Reply:
this is about handling a redirect, that's why ;-)